content/en/docs/security/security-response.md

```diff
@@ -0,0 +1,126 @@
+---
+title: Community Incident Response Guidelines
+weight: 102
+---
+
+Security vulnerabilities should be handled quickly and sometimes privately. The
+primary goal of this process is to reduce the total time users are vulnerable to
+publicly known exploits.
+
+The
+[OpenTelemetry Technical Committee (OTel TC)](https://github.com/open-telemetry/community/blob/main/tech-committee-charter.md)
+and relevant repository maintainers, supported by tooling provided by the
+[SIG Security](https://github.com/open-telemetry/sig-security), are responsible
+for responding to the incident organizing the entire response including internal
+communication and external disclosure.
+
+## Supported Versions
+
+The OTel project provides community support only for the last overall minor
+version: bug fixes are released either as part of the next minor version or as
+an on-demand patch version. Independent of which version is next, all patch
+versions are cumulative, meaning that they represent the state of our `main`
+branch at the moment of the release. For instance, if the latest version is
+0.10.0, bug fixes are released either as part of 0.11.0 or 0.10.1.
+
+Security fixes are given priority and might be enough to cause a new version to
+be released. Each repository is entitled to establish their own complementary
+processes. SIG Security in conjunction with the TC can advise in case
+clarifications are required.
+
+## Disclosures
+
+### Private Disclosure Processes
+
+In order for the vulnerability reports to reach maintainers as soon as possible,
+the preferred way is to use the `Report a vulnerability` button on the
+`Security` tab in the respective GitHub repository. It creates a private
+communication channel between the reporter and the maintainers.
+
+If you are absolutely unable to or have strong reasons not to use GitHub
+reporting workflow, please reach out to the Technical Committee using
+[cncf-opentelemetry-tc@lists.cncf.io](mailto:cncf-opentelemetry-tc@lists.cncf.io)
+and we will provide instruction on how to report the vulnerability using an
+encrypted message, if desired.
+
+### Public Disclosure Processes
+
+If you know of a publicly disclosed security vulnerability please IMMEDIATELY
+email
+[cncf-opentelemetry-tc@lists.cncf.io](mailto:cncf-opentelemetry-tc@lists.cncf.io)
```

Comment:
I wonder if we should have a periodic drill. We could pick one or more random repositories, excluding non-coding ones (oteps, community, semantic-conventions). Something like:

```
gh repo list open-telemetry -l 80 --json name --jq '.[].name' | shuf | head -n 5
opentelemetry-python-contrib
opentelemetry-collector
opentelemetry-rust
opentelemetry-java-examples
opentelemetry-helm-charts
```